src/scaling.jl

```diff
@@ -0,0 +1,104 @@
+"""
+    Scaling <: Transform
+
+Linearly scale the data as `ax + b`, according to some statistics `a` and `b`.
+"""
+abstract type Scaling <: Transform end
+
+"""
+    IdentityScaling
+
+Represents the no-op scaling which simply returns the data it is applied on.
+"""
+struct IdentityScaling <: Scaling end
+# Convenience method
+IdentityScaling(args...; kwargs...) = IdentityScaling()
```

Comment:
I think this is unneccesary?

Reply:
We need it to replace FeatureEngineering directly. I'd like it to be unnecessary, but it's used in some tests for GLMModels. Maybe that itself is unnecessary though.

Reply:
Yeah I would say we can update those tests when we switch